### ISSUE TITLE
Consider Allowing Package From the Package URL

### ISSUE THREAD

**Issue by LukeAbby** [foundryvtt/foundryvtt#10379]
### User Experience

Unimportant in the grand scheme of things but a player sent me a link and asked to add it to Foundry. I was confused for a few minutes because it was a valid URL... but it was actually a URL of the form<https://foundryvtt.com/packages/[name]>.

I think it makes a fair bit of sense to be able to download from a url like that though.

**Comment by aaclayton** [comment]
Yeah, it would be nice to add support for that. It would require an extra API call to request manifest details from the website about which version of the package to install, but that could be done.
